src/borg/archiver.py

```diff
@@ -1233,11 +1233,14 @@ def do_break_lock(self, args, repository):
 
         {now}
 
-            The current local date and time.
+            The current local date and time, by default in ISO-8601 format.
+            You can also supply your own format string, e.g. {now:%y}
 
         {utcnow}
 
-            The current UTC date and time.
+            The current UTC date and time, by default in ISO-8601 format.
+            You can also supply your own format string, e.g. {utcnow:%y}
```

Comment:
We could also give the full %Y-%m-%d_%H:%M:%S example for people who do not like the T separator.